carina-core/src/main/resources/config.properties

```diff
@@ -67,6 +67,7 @@ auto_download_apps=NULL
 auto_download_folder=NULL
 custom_artifacts_folder=NULL
 scroll_to_element_y_offset=120
+element_loading_strategy=BY_PRESENCE_VISIBILITY
```

Comment:
let's rename to BY_PRESENCE_OR_VISIBILITY to satisfy me on 100% :)

Reply:
done